### PR TITLE
Integrated Chrome Built-in AI

### PR DESCRIPTION
**Pull request by Gyan172004**

### CODE REVIEW

File: src/utils/defaultProperties.ts [hunk 1]
```diff
@@ -12,6 +12,7 @@ const LLMWeightsMap = {
   [LLMEngineType.Anthropic]: 0.35,
   [LLMEngineType.Mistral]: 0.13,
   [LLMEngineType.Ollama]: 0.08,
+  [LLMEngineType.ChromeAI]: 0.2,
```
Comment (rootCircle) [inline src/utils/defaultProperties.ts]: change this to a smaller value as local LLMs generally have recent data, so they might hallucinate more!

File: src/docFillerCore/engines/gptEngine.ts [hunk 2]
```diff
@@ -111,6 +114,14 @@ export class LLMEngine {
           apiKey: LLMEngine.apiKeys['anthropicApiKey'] as string,
         });
         break;
+      case LLMEngineType.ChromeAI:
+        LLMEngine.instances[engine] = new ChromeAI({
+          temperature: 0.5,
+          topK: 40,
+          systemPrompt:
+            "You are an expert quiz assistant. For each question: 1) Analyze the question and identify if it's multiple choice, date/time, text, or other format, 2) For multiple choice questions, select the appropriate option(s), 3) For date/time questions, ensure the response is in the correct date format, 4) For text questions, provide a clear concise answer, 5) Always structure your response in the exact JSON format specified in the question. Respond only in English and strictly adhere to the format requirements.",
```
Comment (rootCircle) [inline src/docFillerCore/engines/gptEngine.ts]: We should probably use a generic system prompt, in that way we don't confuse the model about the output data!

**Review by rootCircle — CHANGES_REQUESTED** [review]

File: src/utils/defaultProperties.ts (hunk 1 above)
Comment (Gyan172004) [reply src/utils/defaultProperties.ts]: Yeh, Totally agree, made required changes.

**Review by rootCircle — APPROVED** [review]
LGTM :+1:

**Pull request merged**